fortnite_api/news.py

```diff
@@ -16,39 +52,59 @@ class News:
         The raw data from request. Can be used for saving and re-creating the class.
     """
 
-    def __init__(self, data):
-        self.br = GameModeNews(data.get('br')) if data.get('br') else None
-        self.stw = GameModeNews(data.get('stw')) if data.get('stw') else None
-        self.creative = GameModeNews(data.get('creative')) if data.get('creative') else None
-        self.raw_data = data
+    __slots__: Tuple[str, ...] = ('br', 'stw', 'creative', 'raw_data')
+
+    def __init__(self, *, data: Dict[str, Any], http: HTTPClientT) -> None:
+        _br = data.get('br')
+        self.br: Optional[GameModeNews[HTTPClientT]] = _br and GameModeNews(data=_br, http=http)
+
+        _stw = data.get('stw')
+        self.stw: Optional[GameModeNews[HTTPClientT]] = _stw and GameModeNews(data=_stw, http=http)
 
+        _creative = data.get('creative')
+        self.creative: Optional[GameModeNews[HTTPClientT]] = _creative and GameModeNews(data=_creative, http=http)
```

Comment:
We probably can remove this, since creative news have been discontinued since later 2021. This field just exists as legacy